### ISSUE TITLE
(Some) kwargs don't seem to be properly handled by requests_cache.install_cache()

### ISSUE THREAD

**Issue by pfalcon** [requests-cache/requests-cache#890]
### The problem
According to https://requests-cache.readthedocs.io/en/stable/modules/requests_cache.patcher.html#requests_cache.patcher.install_cache :
```
requests_cache.patcher.install_cache(cache_name='http_cache', backend=None, session_factory=<class 'requests_cache.session.CachedSession'>, **kwargs)

Accepts all the same parameters as CachedSession.
```

However, `expire_after` param in:
```
requests_cache.install_cache(
    '.api_cache',
    backend='filesystem',
    expire_after=1800,
)
```

doesn't seem to work, responses are forever-cached as without it. I had to workaround this with cumbersome:
```
session = requests_cache.CachedSession(
    '.api_cache',
    backend='filesystem',
    expire_after=1800,
)
requests_cache.install_cache(session)
```

### Expected behavior
See above.

### Steps to reproduce the behavior
See above.

### Workarounds
See above.

### Environment
- requests-cache version: 1.1.0
- Python version: 3.10
- Platform: Ubuntu 22.04

**Comment by JWCook** [comment]
> doesn't seem to work, responses are forever-cached as without it. 

Can you provide a complete code example that shows the problem? In your first example, expiration appears to be set correctly:

```py
import requests
import requests_cache

requests_cache.install_cache(
    '.api_cache',
    backend='filesystem',
    expire_after=1800,
)
r = requests.get('https://github.com/requests-cache/requests-cache')
print(r.expires)
```

If you previously cached the same request with no expiration, you will need to clear the cache first.

**Comment by pfalcon** [comment]
@JWCook: First of all, I appreciate prompt, detailed, and patient response, to what clearly looks like a confused user's noise even to myself ;-).

> Is it possible that you previously cached the same request with no expiration? 

Arghh, so that's indeed the root cause of my confusion! But then it seems like pretty impeccable process:
* Yesterday I tried to add request_cache to my script, everything worked out of the box by just following quick start guide, great!
* Today, running the same script, I see that I don't get a new data. Fortunately, I remember that there's easy option to add expiration timeout, add it, but it doesn't work.
* Without reading docs, I'm trying to munge it all "by intuition", leading to only more confusion with #889.

So yes, I see that with `expire_after=`, there's an `expires` key being added to cache item metadata. I would exclaim "But why?!", but doing thinking exercise, I guess it's because HTTP's cache control headers use that absolute timestamp format, and then to have common handling of expiration, `expire_after=` uses just the same "store-per-item" functionality.

So, I guess it makes no sense to argue that `expire_after=`'s expiration shouldn't be store in teh cache, but just calculated based on original request timestamp. But then, as I try to exemplify above, this behavior may be not obvious to an end user at all, and quite confusing. Perhaps I missed some FAQ which explains that, but I'd suggest to mention this situation (need to regenerate cache if `expire_after=` is added later) right in its description, even in https://github.com/requests-cache/requests-cache#headers-and-expiration (a lot of people probably won't look beyond README), and definitely in https://requests-cache.readthedocs.io/en/stable/user_guide/expiration.html

What that page says now:

> The simplest option is to initialize the cache with an expire_after value, which will apply to all reponses:

Is way too implicit. Yes, now knowing about this behavior, I agree that the notice above hints at it, but starting just from that phrase, make an entailment of the need to regenerate the cache if initially the option wasn't used - not easy at all.

Closing otherwise, thanks again!
